src/helper_functions.jl

```diff
@@ -228,3 +230,122 @@ function check_method_ambiguity_binary(op;
 #     result &= !has_other_problems # also report other errors
     return result
 end
+
+"""
+    @commutative_neutral(SET, NEUT)
+
+Creates functions to make a set type behave commutative with a given neutral
+element set type.
+
+### Input
+
+- `SET`  -- set type
+- `NEUT` -- set type of the neutral element
+
+### Output
+
+Three function definitions.
+
+### Examples
+
+`@commutative_neutral(ConvexHull, EmptySet)` creates the following functions:
```

Comment:
you can more synthetically write the equations, eg. "it creates the following functions for any set ``X``: ``CH(X, ∅) = CH( ∅, X) = X``  and ``CH(X, ∅) = X`` .

Reply:
`∅` might be misleading, but I get your point.